src/dark-mode-toggle.mjs

```diff
@@ -98,10 +98,10 @@ form {
 fieldset {
   border: none;
   margin: 0;
-  padding-block-start: 0.25rem;
-  padding-block-end: 0.25rem;
-  padding-inline-start: 0.25rem;
-  padding-inline-end: 0.25rem;
+  padding-block-start: .25rem;
+  padding-block-end: .25rem;
+  padding-inline-start: .25rem;
+  padding-inline-end: .25rem;
```

Comment:
Not a big fan personally (a CSS minifier can do this for me), but turns out it's [in our style guide](https://google.github.io/styleguide/htmlcssguide.html#Leading_0s) even. Thanks.

Reply:
I definitely see what you're saying. It _does_ feel like something a build tool should take care of. OTOH, until we have such a tool...

Reply:
Again something CSS Modules will enable…